config/initializers/ldap_instrumentation.rb

```diff
@@ -1,8 +1,8 @@
+require 'foreman/telemetry'
+
 # Debug logging from net-ldap and ldap_fluff events sent via ActiveSupport::Notifications
 module Foreman
   class LdapSubscriber < ActiveSupport::LogSubscriber
-    include Foreman::TelemetryHelper
```

Comment:
I believe this `include` is needed and `Foreman::TelemetryHelper` should be autoloaded.
Tho I also believe there is no need for this subscriber to be registred prior `after_initialize`.